modules/pagereader.py

```diff
@@ -44,10 +44,13 @@ def read_first_page(site, extension=False):
             attempts_left -= 1
             err = e
 
-    if err == HTTPError:
-        raise("There has been an HTTP error after three attempts.")
-    if err == ConnectionError:
-        raise("There has been a connection error after three attempts.")
+    if type(err) == HTTPError:
+        print ("There has been an HTTP error after three attempts.")
+        exit (2)
```

Comment:
Add a comment explaining what the exit code 2 does.

Reply:
changed exit code to 1 
as per 
https://docs.python.org/3/library/sys.html#sys.exit